MessengerOnFileForLinux/Common/test/TimeManagmentTest.cpp

```diff
@@ -22,7 +22,21 @@ TEST(ConvertTimeToString, TimeFormatIsCorrect)
 
 TEST(ConvertTimeToString, TimeValueIsCorrect)
 {
-    const auto unixEpochTimeAsString = TimeManagment::convertTimeToString(std::time_t{});
-    std::string expectedTimeValue = "1970-01-01 01:00:00";
-    EXPECT_EQ(unixEpochTimeAsString, expectedTimeValue);
+    constexpr auto day = 9;
```

Comment:
misuse auto, line 26-27

Reply:
auto should not be used for simply types as int, char, string due to cannot easy check what time is it and easy way for mistake. For example for(auto i = 0; i < s.size(); ++i), uint or int? which it will be and which it should be?

Reply:
You're right. What do you thing about  constexpr auto day = 9u;

Reply:
ok